stairway/src/test/java/bio/terra/stairway/impl/FilterTest.java

```diff
@@ -38,13 +38,13 @@ public void predicateTest() throws Exception {
   }
 
   private void testPredicate(FlightFilter filter, int index, String operand) {
-    String flightCompareSql = String.format("F.afield %s :ff%d", operand, index + 1);
+    String flightCompareSql = String.format("F.afield %s :ff%d", operand, 1);
```

Comment:
With this change the parameter IDs aren't incremented across queries; each instance of `FlightFilterAccess` has its own counter for IDs. It looks like each query is run independently so this should be OK to do. If it isn't, a global parameter counter could be used to ensure that IDs are unique across all queries.